experimental/list_formatter/src/list_formatter.rs

```diff
@@ -53,21 +86,22 @@ impl ListFormatter {
         }
     }
 
-    pub const fn element() -> Part {
-        Part {
-            category: "list",
-            value: "element",
-        }
-    }
+    /// The [`Part`] used by [`List`] to mark the part of the string that is an element.
+    pub const ELEMENT: Part = Part {
+        category: "list",
+        value: "element",
+    };
 
-    pub const fn literal() -> Part {
-        Part {
-            category: "list",
-            value: "literal",
-        }
-    }
+    /// The [`Part`] used by [`List`] to mark the part of the string that is a list literal,
+    /// such as ", " or " and ".
+    pub const LITERAL: Part = Part {
+        category: "list",
+        value: "literal",
+    };
```

Comment:
I'd rather put these on `List`, but because it's generic they couldn't be referenced as `List::LITERAL`

Reply:
It might actually be a good idea to introduce a top-level module named something like `parts` where you put all your parts constants.  We have precedent for this already with data keys in `provider::keys`.

Reply:
Ok. But I'd rather put them on List :(